src/providers/seventv/SeventvEmoteCache.cpp

```diff
@@ -0,0 +1,33 @@
+#include "SeventvEmoteCache.hpp"
```

Comment:
Assuming we're not losing 99% performance or something, I'd prefer a simpler uncached version to start with, will make it easier to understand what the PR does and keeps it as minimal as possible